lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs

```diff
@@ -336,6 +395,146 @@
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false}}
+          <div class="row">
+            <button
+              class="btn btn-sm bg-primary inline-block mb-5"
+              style="margin-top: -5px;" type="button" {{action "addYAML"}}
+            >
+              {{t "clusterNew.ociocne.addons.yaml.addNew"}}
+            </button>
+          </div>
+          {{#each yamls as |yaml|}}
+            <div class="row">
+              <div class="col span-4">
+                {{input-yaml
+                  showUpload=true
+                  showDownload=true
+                  canChangeName=false
+                  value=yaml.body
+                  autoResize=true
+                  loading=loading
+                }}
+              </div>
+            </div>
+
+            <button
+              class="btn btn-sm bg-primary inline-block mb-5"
+              style="margin-top: -5px;" type="button" {{action "deleteYAML" yaml}}
+            >
+              {{t "clusterNew.ociocne.addons.yaml.delete"}}
+            </button>
+          {{/each}}
+        {{/accordion-list-item}}
+        {{#accordion-list-item title=(t "clusterNew.ociocne.networking.title")
+                               detail=(t "clusterNew.ociocne.networking.detail")
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false}}
+          {{#if (eq vcnCreationMode "Existing")}}
+            <div class="row">
+              <div class="col span-4 mb-0">
+                <label>
+                  {{t 'clusterNew.ociocne.networking.clusterCidr.title'}}{{field-required}}
+                </label>
+                {{input type="text" name="cluster-cidr" classNames="form-control" value=config.clusterCidr}}
+                <p class="help-block">
+                  {{t 'clusterNew.ociocne.networking.clusterCidr.help'}}
+                </p>
+              </div>
+              <div class="col span-4 mb-0">
+                <label for="etcd-tag">
+                  {{t 'clusterNew.ociocne.networking.podCidr.title'}}{{field-required}}
+                </label>
+                {{input type="text" name="pod-cidr" classNames="form-control" value=config.podCidr}}
+                <p class="help-block">
+                  {{t 'clusterNew.ociocne.networking.podCidr.help'}}
+                </p>
+              </div>
+            </div>
+          {{/if}}
+          <div class="row">
+            <div class="col span-4 mb-0">
+              <label>
+                {{t 'clusterNew.ociocne.networking.proxyEndpoint.title'}}
+              </label>
+              {{input type="text" name="proxy-endpoint" classNames="form-control" placeholder=(t 'clusterNew.ociocne.networking.proxyEndpoint.placeholder') value=config.proxyEndpoint}}
```

Comment:
Do we want to keep the "proxy endpoint" field in the advanced settings > cluster networking subsection? It's not mentioned in the doc